napari/tests/test_viewer.py

```diff
@@ -45,7 +45,7 @@ def test_no_qt_loop():
     magic.
     """
     with pytest.raises(RuntimeError):
-        viewer = Viewer()
+        viewer = Viewer()  # noqa
```

Comment:
@tlambert03 Can you drop the `# noqa` here and just have:
```python
with pytest.raises(Runtime error):
    Viewer()
```

Reply:
Another alternative with me hoping against hope: does `_ = Viewer()` get ignored?

Reply:
@bjlittle, yes you could.  But i was again opting for the more explicit option there, since @jni wrote that code and I didn't risk messing up the test.  @jni, you're in luck: flake8 will ignore unused underscore variables.  I think that's the best solution here.

Reply:
YASSSS! Let's do that where needed!